src/aws/lambda-mediapackage/src/utils/setRecordingSliceManifestKey/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Get the record slices states for a video, using hmac with a shared secret
+ * to sign the object key and provide authorization for the request.
+ */
+const { computeSignature, sendRequest } = require('update-state/utils');
+
+const { RECORDING_SLICES_MANIFEST_ENDPOINT, SHARED_SECRET } = process.env;
+const DISABLE_SSL_VALIDATION = JSON.parse(process.env.DISABLE_SSL_VALIDATION);
+
+module.exports = async (key, harvestJobId, manifestKey = {}) => {
```

Comment:
```suggestion
module.exports = async (video_id, harvestJobId, manifestKey = {}) => {
```

Same comment as before, you can choose a more relevant name here, this function will only be used in a video context